inc/html.class.php

```diff
@@ -6806,9 +6806,14 @@ private static function displayMainMenu($full, $options = []) {
          }
 
          //  Tickets
-         if (Session::haveRight("ticket", CREATE)
+         if ((Session::haveRight("ticket", CREATE)
             || Session::haveRight("ticket", Ticket::READMY)
-            || Session::haveRight("followup", ITILFollowup::SEEPUBLIC)) {
+            || Session::haveRight("followup", ITILFollowup::SEEPUBLIC))
+            && !(
+               Plugin::isPluginActive('formcreator')
+               && \Session::getCurrentInterface() == "helpdesk"
+            )
+         ) {
```

Comment:
Is this ok ? Checking for enabled plugin in core.
This is done to avoid removing the link using js/css (:nauseated_face:).

Reply:
Not OK for me...